### PR TITLE
Remove old code

### PR DESCRIPTION
**Pull request by mimse**
Known issues with mock data

Filters on users and permission do not work.

### CODE REVIEW

File: issues_raccons.json
Comment (dzhavat) [inline issues_raccons.json:1]: question: is this file necessary?

File: libs/dh/admin/feature-permissions/project.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  "sourceRoot": "libs/dh/admin/feature-permissions/src",
+  "sourceRoot": "libs/dh/admin/feature-permissions-new/src",
   "prefix": "dh",
   "targets": {
     "test": {
       "executor": "@nx/jest:jest",
       "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
       "options": {
-        "jestConfig": "libs/dh/admin/feature-permissions/jest.config.ts"
+        "jestConfig": "libs/dh/admin/feature-permissions-new/jest.config.ts"
```
Comment (dzhavat) [inline libs/dh/admin/feature-permissions/project.json:12]: suggestion: the `-new` suffix is not necessary anymore.

File: libs/dh/admin/feature-user-management/project.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-  "sourceRoot": "libs/dh/admin/feature-user-management/src",
+  "sourceRoot": "libs/dh/admin/feature-user-management-new/src",
   "prefix": "dh",
   "targets": {
     "test": {
       "executor": "@nx/jest:jest",
       "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
       "options": {
-        "jestConfig": "libs/dh/admin/feature-user-management/jest.config.ts"
+        "jestConfig": "libs/dh/admin/feature-user-management-new/jest.config.ts"
```
Comment (dzhavat) [inline libs/dh/admin/feature-user-management/project.json:12]: suggestion: the `-new` suffix is not needed anymore

File: libs/dh/admin/feature-user-management/src/lib/details/details.component.ts [hunk 3]
```diff
@@ -17,144 +17,122 @@
...
     effect(() => {
-      const userId = this.userId();
-      if (!userId) return;
-      this.selectedUserQuery.refetch({ id: userId });
+      const id = this.id();
+      const drawer = this.drawer();
+      if (!id || !drawer) return;
+      this.drawer().open();
+      this.selectedUserQuery.refetch({ id });
+      drawer.open();
```
Comment (dzhavat) [inline libs/dh/admin/feature-user-management/src/lib/details/details.component.ts:121]: issue: the `drawer.open()` is called twice.

File: libs/dh/admin/feature-user-roles/project.json [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
-  "sourceRoot": "libs/dh/admin/feature-user-roles/src",
+  "sourceRoot": "libs/dh/admin/feature-user-roles-new/src",
   "prefix": "dh",
   "targets": {
     "test": {
       "executor": "@nx/jest:jest",
       "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
       "options": {
-        "jestConfig": "libs/dh/admin/feature-user-roles/jest.config.ts"
+        "jestConfig": "libs/dh/admin/feature-user-roles-new/jest.config.ts"
```
Comment (dzhavat) [inline libs/dh/admin/feature-user-roles/project.json:12]: suggestion: the `-new` suffix is not needed anymore

**Review by dzhavat — COMMENTED** [review]
praise: looks good 💯. Just a few minor comments.

File: issues_raccons.json
Comment (mimse) [reply issues_raccons.json:1]: Nooo, could not have been commited, me bad

File: libs/dh/admin/feature-user-management/src/lib/details/details.component.ts (hunk 3 above)
Comment (mimse) [reply libs/dh/admin/feature-user-management/src/lib/details/details.component.ts:121]: Nice find!

**Review by dzhavat — APPROVED** [review]
